static/js/unit_maps.js

```diff
@@ -15,7 +20,9 @@ var UnitMaps = {
     var features = L.featureGroup();
 
     $.each(locations.features, function (_, location) {
-      if (['node', 'point'].indexOf(location.properties.feature_type) >= 0) {
+      const featureType = UnitMaps.getFeatureType(location)
+
+      if (['node', 'point'].includes(featureType)) {
```

Comment:
also refactored to use `includes` instead of `indexOf`. see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes